### PR TITLE
chore(package): update rxjs dev and peer to version 6.x

### PR DESCRIPTION
**Pull request by ericadamski**
This pull request is related to #85.

Currently all that is changed are the import statements mentioned in https://github.com/ReactiveX/rxjs/blob/master/MIGRATION.md#import-paths.

The README.md has not been updated yet to mention the support for RxJS version 6.x

I was not able to run `npm run lint:staged` running `npm run lint` is fine thought. Because of this I was also not able to run `npm run commit`.

### CODE REVIEW

File: src/scheduler/TestScheduler.ts [hunk 1]
```diff
@@ -106,7 +103,7 @@ class TestScheduler extends VirtualTimeScheduler {
 
     const messages = Array.isArray(marbleValue)
       ? marbleValue
-      : parseObservableMarble(marbleValue, value, error, false, this.frameTimeFactor, this._maxFrame) as any;
+      : (parseObservableMarble(marbleValue, value, error, false, this.frameTimeFactor, this._maxFrame) as any);
```
Comment (kwonoj) [inline src/scheduler/TestScheduler.ts]: does parentheses needed here and line 89?

File: package.json [hunk 2]
```diff
@@ -66,19 +66,20 @@
     "cz-conventional-changelog": "2.1.0",
     "husky": "^0.14.3",
     "jest": "^22.0.4",
+    "jest-cli": "^22.4.3",
```
Comment (kwonoj) [inline package.json]: installing jest does / should not require separate install to jest-cli. (it's dependency of `jest` itself). Please revert it back.

File: spec/marbles/parseObservableMarble-spec.ts [hunk 3]
```diff
@@ -30,7 +30,12 @@ describe('parseObservableMarble', () => {
   it('should correctly parse falsy timeframe value', () => {
     const marble = '--a-b-c-d';
 
-    const messages = parseObservableMarble(marble, { a: null, b: false, c: 0, d: undefined });
+    const messages = parseObservableMarble(marble, {
```
Comment (kwonoj) [inline spec/marbles/parseObservableMarble-spec.ts]: please preserve formatting.

**Comment by ericadamski** [comment]
Should have fix the things requested <3 thank you!

**Pull request merged**